src/lib/legrand.ts

```diff
@@ -0,0 +1,112 @@
+import {Fz, Tz, OnEvent} from '../lib/types';
+import * as exposes from './exposes';
+import * as utils from '../lib/utils';
+const e = exposes.presets;
+const ea = exposes.access;
+
+const legrandOptions = {manufacturerCode: 4129, disableDefaultResponse: true};
+
+const shutterCalibrationModes = {
+    'Classic (NLLV)': {ID: 0, onlyNLLV: true},
+    'Specific (NLLV)': {ID: 1, onlyNLLV: true},
+    'Up/Down/Stop': {ID: 2, onlyNLLV: false},
+    'Temporal': {ID: 3, onlyNLLV: false},
+    'Venetian (BSO)': {ID: 4, onlyNLLV: false},
```

Comment:
Last thing before this can be merged, could you make these `snake_case`? e.g. `classic_nllv`, `venetian_bso`

Reply:
I need those strings in the frontend, but let me change the structure somewhat. I assume you with to avoid having strings containing special chars. as keys right ?

Reply:
I meant the values itself (the structure is fine). No special characters indeed, no spaces and only lower case, e.g. `Venetian (BSO)` should become `venetion_bso`, `Up/Down/Stop` -> `up_down_stop`.

Reply:
Oh okay, sure. Let me change it then.
Just figured that it would look better as "text", like so:

<img width="1319" alt="grafik" src="https://github.com/Koenkk/zigbee-herdsman-converters/assets/141126144/1a7df17c-0ae8-4385-a51b-27a047739f36">

Reply:
PR updated accordingly.

<img width="1431" alt="grafik" src="https://github.com/Koenkk/zigbee-herdsman-converters/assets/141126144/0660c33c-278c-4fbb-a793-3c102d2cffec">

Reply:
Not all people use the frontend, e.g. when you publish directly to MQTT, you don't want to deal with special chars.

Reply:
Fair point, thanks @Koenkk